packages/core/src/sdk/ui/useScreenResize.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react'
+
+const MAX_MOBILE_WIDTH = 420
```

Comment:
question: I was wondering if we could just update the `phonemid` to `420px`. (As far as I checked, we are not using either `phonemid` nor `phone` breaking points.)

@renatamottam  WDYT?

https://github.com/vtex/faststore/blob/397a6ee1653fd5c8b42bd46429be651a95646ccc/packages/ui/src/styles/base/utilities.scss#L14-L20

Reply:
I think the 375px would cover most cases:

<img width="1284" alt="Screenshot 2024-10-11 at 10 46 50" src="https://github.com/user-attachments/assets/1b1950cb-12b6-4c83-8616-3c250c7c5385">
uld 
https://gs.statcounter.com/screen-resolution-stats/mobile/worldwide

Reply:
it does make sense although we are not using it.

@lucasfp13 I think we can keep it, and add the comment explaining why we are setting as `420` (actually is 412 hahaha) as I mentioned previously.